Doc/library/stdtypes.rst

```diff
@@ -394,10 +394,13 @@ Bitwise Operations on Integer Types
    pair: bitwise; operations
    pair: shifting; operations
    pair: masking; operations
+   pair: inversion; operations
```

Comment:
I think this index entry is not needed. `~` is a bitwise operation. There are no separate index entries for bitwise conjunction and disjunction.